### PR TITLE
Updated README with suggestions from Björn

### PR DESCRIPTION
**Pull request by bwplotka**
Suggestions were made on https://github.com/prometheus/docs/pull/2303

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,38 +1,39 @@
...
+3. An automatic formatter is enabled in the repository. Use `make` locally to trigger the formatting of all markdown documents (requires working Go environment). Use `make check` to check all links (will be done by the CI pipeline, too).
+4. The design is accepted if the PR is merged into this repository. It’s ok to eventually decide to reject the proposal and close the PR with meaningful reasons for why it was rejected.
+   1. If more eyes are needed or no consensus was made: Propose and announce your idea on the [Prometheus DevSummit](https://docs.google.com/document/d/11LC3wJcVk00l8w5P3oLQ-m3Y37iom6INAMEu2ZAGIIE/edit) or mailing list to gather more information. You are welcome to start working on the design document before a bigger discussion—it is often easier to discuss with prior details provided. Be prepared that the idea might be rejected later. Still, the record of the document in the Pull Request is valuable even in a rejected state to inform about past decisions and opportunities considered.
```
Comment (beorn7) [inline README.md:35]: Suggestion for the first sentence:

Propose your idea as an agenda item for the [Prometheus DevSummit](https://docs.google.com/document/d/11LC3wJcVk00l8w5P3oLQ-m3Y37iom6INAMEu2ZAGIIE/edit) or announce it to the [developer mailing list](https://groups.google.com/forum/#!forum/prometheus-developers) to gather more information.

**Review by beorn7 — CHANGES_REQUESTED** [review]
Looks good. Some minor suggestions.

File: README.md [hunk 2]
```diff
@@ -1,38 +1,39 @@
...
+   1. If you prefer Google Docs to any other collaboration tool, feel free to use it in the initial state. We recommend the [Open Source Design document Template](https://docs.google.com/document/d/1zeElxolajNyGUB8J6aDXwxngHynh4iOuEzy3ylLc72U/edit#). However, the approval process will only happen officially in the Pull Request.
+3. An automatic formatter is enabled in the repository. Use `make` locally to trigger the formatting of all markdown documents (requires working Go environment). Use `make check` to check all links (will be done by the CI pipeline, too).
+4. The design is accepted if the PR is merged into this repository. It’s ok to eventually decide to reject the proposal and close the PR with meaningful reasons for why it was rejected.
```
[the remark this answers is not in the document]
Comment (bwplotka) [reply README.md:34]: love it, thanks!

**Comment by bwplotka** [comment]
Addressed all, PTAL (:

File: README.md [hunk 3]
```diff
@@ -1,38 +1,39 @@
 # proposals
```
Comment (beorn7) [inline README.md:1]: Just noticed this: Shouldn't this be capitalized?

File: README.md [hunk 4]
```diff
@@ -1,38 +1,39 @@
...
+   1. If you prefer Google Docs to any other collaboration tool, feel free to use it in the initial state. We recommend the [Open Source Design document Template](https://docs.google.com/document/d/1zeElxolajNyGUB8J6aDXwxngHynh4iOuEzy3ylLc72U/edit#). However, the approval process will only happen officially in the Pull Request.
+3. An automatic formatter is enabled in the repository. Use `make` locally to trigger the formatting of all markdown documents (requires a working Go environment). Use `make check` to check all links (will be done by the CI pipeline, too).
+4. After a sufficient amount of discussion, the Prometheus team will try to reach a consensus of accepting or rejecting the proposal. In the former case, the PR gets merged. In the latter case, the PR get closed with meaningful reasons why the proposal was rejected.
```
Comment (beorn7) [inline README.md:34]: the PR get closed → the PR gets closed

**Review by beorn7 — APPROVED** [review]
Modulo a few other tiny things I have seen now.

File: README.md (hunk 3 above)
Comment (bwplotka) [reply README.md:1]: not really as repo name is proposals, but not a big deal - will captilize.

**Pull request merged**